src/lib/containers/download_list_v2/MeetAccessRequirementCard.tsx

```diff
@@ -0,0 +1,113 @@
+
+export const MeetAccessRequirementCard:React.FunctionComponent<MeetAccessRequirementCardProps> = (
+  {
+    accessRequirementId,
+    count,
+  }: MeetAccessRequirementCardProps
+) => {
+  const handleError = useErrorHandler()
+  const {
+    data,
+    isFetching,
+    isError,
+    error: newError,
+  } = useGetAccessRequirement(accessRequirementId)
+  const ar:AccessRequirement = data as AccessRequirement
+  useEffect(() => {
+    if (isError && newError) {
+      handleError(toError(newError))
+    }
+  }, [isError, newError, handleError])
+  const [isShowingAccessRequirement, setIsShowingAccessRequirement] = useState<boolean>(false)
+  let content = <></>
+  if (!isError && !isFetching) {
+    let title:string|undefined = undefined
+    let iconType = ''
+    let description = ''
+    switch(ar.concreteType) {
+      case SUPPORTED_ACCESS_REQUIREMENTS.TermsOfUseAccessRequirement:
+        title = TERMS_OF_USE_TITLE
+        iconType = EASY_DIFFICULTY
+        description = ar.description ?? ''
+        break;
+      case SUPPORTED_ACCESS_REQUIREMENTS.SelfSignAccessRequirement: {
+        title = SELF_SIGN_TITLE
+        const selfSignAR:SelfSignAccessRequirement = ar as SelfSignAccessRequirement
+        if (selfSignAR.isValidatedProfileRequired) {
+          iconType = VARIABLE_DIFFICULTY
+        } else if (selfSignAR.isCertifiedUserRequired) {
+          iconType = MEDIUM_DIFFICULTY
+        } else {
+          iconType = EASY_DIFFICULTY
+        }
+        description = ar.description ?? ''
+        break;
+      }
+      case SUPPORTED_ACCESS_REQUIREMENTS.ManagedACTAccessRequirement:
+      case SUPPORTED_ACCESS_REQUIREMENTS.ACTAccessRequirement:
+        title = ACT_TITLE
+        iconType = VARIABLE_DIFFICULTY
+        description = ar.description ?? ''
+        break;
+      case 'org.sagebionetworks.repo.model.LockAccessRequirement':
+        title = LOCK_TITLE
+        iconType = VARIABLE_DIFFICULTY
+        description = 'Access restricted pending review by Synapse Access and Compliance Team.'
+        break;
+    }
+    if (title) {
+      content = <div className="MeetAccessRequirementCard actionRequiredCard">
+      <Icon type={iconType} />
+      <div className="metadata">
+        <div className="title">{title}</div>
+        <div className="fileCount">{count} File(s)</div>
+        <div className="description">
+          {description}
+        </div>
+      </div>
+      <div className="startButtonContainer">
+        <Button className="startButton" variant="secondary"
+          onClick={()=>setIsShowingAccessRequirement(true)}
+        >Start</Button>
+      </div>
+    </div>
+    }
+  }
+
+  return (
+    <>
+      {!isError && !isFetching && content}
+      {isFetching && (
+        <div className="placeholder">
+          <SynapseSpinner size={30} />
+        </div>
+      )}
+      {isShowingAccessRequirement && (
+        <AccessRequirementList entityId={ar.subjectIds[0].id}
```

Comment:
Clicking on the START button will show the AccessRequirementList dialog that contains that single Access Requirement.  An Access Requirement is associated to a number of subjects, but I simply pick the first entity (since I'm not sure what file in the download list is associated to this AR).